Friendly-plans/app/src/main/java/pg/autyzm/friendly_plans/validation/TaskValidation.java

```diff
@@ -58,10 +58,10 @@ private boolean checkUpdateNameDoesNotExist(Long taskId, String name) {
 
     public ValidationResult isDurationValid(String duration) {
         ValidationResult validationResult = isStringEmpty(duration);
-        if (validationResult.getValidationStatus().equals(ValidationStatus.INVALID)) {
-            return validationResult;
+        if (validationResult.getValidationStatus().equals(ValidationStatus.VALID)) {
```

Comment:
I thinks It's a little confusing, when string is empty it is still being checked as invalid but it will always return VALID. Could you please change it to validate only 
 the condition : if there is any value it should be a number format ? I think some tests will have to be updated with it also.

Reply:
Sure, honestly I find that a bit confusing as well.
So do you suggest adding a new metod isDurationValid to Validation class? Something like with name validation? Do I understand correctly?
![image](https://user-images.githubusercontent.com/32238789/40278155-fe08e952-5c2b-11e8-8143-342e1f76dd2f.png)
![image](https://user-images.githubusercontent.com/32238789/40278166-45368a14-5c2c-11e8-93c5-bb7ad5e5ab06.png)

Reply:
Okay, I see, so you suggest rewriting isNameValid so that it does not use isStringEmpty, and than creating isDurationValid in a similar fashion. 
Is that right?

Reply:
Yes, isStringEmpty is about checking if string is empty and mark it as invalid, when in this case it is valid. So the function should not be used here.